added info on how to start influx (#4113)
influxdata/docs-v2: content/influxdb/v2.1/get-started.md

- [Visualize data](#visualize-data)
- [Monitor and alert](#monitor-and-alert)

> **NOTE:** You **must** start the influxd daemon in order to start working with InfluxDB.  See [InfluxDB service](/influxdb/v2.1/reference/cli/influxd/). For information on the command line interface (CLI) see [InfluxDB command line interface](/influxdb/v2.1/reference/cli/influx/).  Once you've started the influxd daemon, use `localhost:8086` to log in to your instance. 

Reviewer: ```suggestion
*Note:** To run InfluxDB, start the `influxd` daemon ([InfluxDB service](/influxdb/v2.1/reference/cli/influxd/)) using the [InfluxDB command line interface](/influxdb/v2.1/reference/cli/influx/). Once you've started the `influxd` daemon, use `localhost:8086` to log in to your instance.
```

> To start Influx do the following:
>   1. Open a terminal.
>   2. Type influxd in the command line.

Reviewer: ```suggestion
To start InfluxDB, do the following:
  1. Open a terminal.
  2. Type `influxd` in the command line.
```

```bash

> influxd

Reviewer: to be consistent in our code examples, we typically use `sh` in lieu of `bash`
(unless there's a technical distinction to be made (eg, you must use `bash` bc `sh` doesn't support that functionality).

Author: thanks Kelly
